packages/owlbot-bootstrapper/cli/test/run-trigger.test.ts

```diff
@@ -154,4 +162,33 @@ describe('tests running build trigger', () => {
       }
     );
   });
+
+  describe('get mono repo name and org from ssh address', () => {
```

Comment:
nit: I would just name this describe block the exported function name being tested

This lets you easily run the subset of tests:

```
npm t -- -g 'parseRepoNameAndOrg'
```